toto/matchrule_validators.py

```diff
@@ -32,22 +32,24 @@ def _validate_match_rule(keywords):
     new_target = target
 
   elif len(keywords) == 7:
-    rule, artifact, target, from_keyword, step, as_keyword, new_target = keywords
+    (rule, artifact, target, as_keyword, new_target, from_keyword,
```

Comment:
You call the left path argument "target" and the right path argument "new_target" which is different from the existing naming scheme in verifylib.
```python
["MATCH", "MATERIAL", "<path pattern>", "FROM", "<step name>"]
["MATCH", "PRODUCT", "<path pattern>", "FROM", "<step name>"]
["MATCH", "MATERIAL", "<path pattern>", "AS", "<target path pattern>", "FROM", "<step name>"]
["MATCH", "PRODUCT", "<path pattern>", "AS", "<target path pattern>", "FROM", "<step name>"]
```
I'm not happy with either of them but I think we should stay consistent, or at least add a `# FIXME` comment.